manifests/service.pp

```diff
@@ -5,30 +5,30 @@
 # == Authors
 #
 # * Trevor Vaughan <mailto:tvaughan@onyxpoint.com>
-#
+
 class sssd::service {
 
-  file { '/etc/init.d/sssd':
+  file { $sssd::sssd_service_path:
     ensure  => 'file',
     owner   => 'root',
     group   => 'root',
     mode    => '0754',
     seltype => 'sssd_initrc_exec_t',
     source  => 'puppet:///modules/sssd/sssd.sysinit',
-    notify  => Service['sssd']
+    notify  => Service[$sssd::sssd_service_name]
   }
 
-  service { 'nscd':
+  service { $sssd::nscd_service_name:
```

Comment:
The goal here was to keep the exact same functionality that existed before. This PR is for refactoring to data in modules, not fixing other problems with the module. There should be a different ticket for this if you want it fixed.